src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,93 @@
+$blue-color: #00acdc;
+$white-color: #fff;
+$text-color: #060b35;
+
+.card {
+  box-sizing: border-box;
+  background-color: $white-color;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  font-family: "Roboto", sans-serif;
```

Comment:
Consider using the font family for the entire body to keep elements on the page inherit this style